internal/cmd/policy/policy.go

```diff
@@ -24,6 +25,7 @@ func New() *cobra.Command {
 	cmd.AddCommand(addkey.New(o))
 	cmd.AddCommand(addrule.New(o))
 	cmd.AddCommand(removerule.New(o))
+	cmd.AddCommand(listrules.New(o))
```

Comment:
nit: can you place this line above `removerule` to be alphabetical? I'm aware this isn't consistent right now but as we add subcommands, it'll be helpful to order them upfront.

Reply:
Moved